code/modules/mob/living/living.dm

```diff
@@ -650,38 +650,38 @@
 /mob/living/proc/makeTrail(turf/T)
```

Comment:
Also here, I would rename `T` to make it more undestandable. Is this a turf we making trail to?
Or is it a turf we making trail from. 1 letter vars are OK only in the cases where it means nothing, like iterator in loop,
but here - it's logic heave

Reply:
@DGamerL please, do the renaming here and I'm good with you refactor :D

Reply:
Yeah I was still quickly looking at what I should name it, will give an update on it tonight!